templates/components/itilobject/fields/status.html.twig

```diff
@@ -70,3 +70,8 @@
    __('Status'),
    field_options
 ) }}
+
+{% if not item.isNewItem() %}
+   <input type="hidden" name="global_validation" value="{{ item.fields.global_validation }}">
```

Comment:
This field is not supposed to be present in the form. If it has to be set for rule engine only, it should be done inside the `Ticket::fillInputForBusinessRules()` method.